.github/workflows/docker-build.yml

```diff
@@ -344,7 +344,8 @@ jobs:
           target: ${{ inputs.target }}
           provenance: ${{ inputs.provenance }}
           sbom: ${{ inputs.with_sbom }}
-      - name: Save Image cache
+      - if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop' 
```

Comment:
Do we not need for release branches as well?

Reply:
I have added the release branch aswell.